pkg/netpol/connlist/connlist.go

```diff
@@ -229,6 +271,8 @@ type Peer2PeerConnection interface {
 	ProtocolsAndPorts() map[v1.Protocol][]eval.PortRange
 	// String returns a string representation of the connection object
 	String() string
+	TxtFormat() string
```

Comment:
Consider using an external formatter/function accepting a connection and returning a string (kind of like the way Marshallers work - you may even be able to use the default JSON marshaller for this...)

Reply:
moved these to the new file too, tried to avoid code duplication by using the "json" struct for both formats